src/summary/documentation/dimensions-card.resource.ts

```diff
@@ -17,15 +17,17 @@ function getDimensionsObservations(patientId: string) {
   ).pipe(
     map(({ data }) => data["entry"]),
     map(entries => {
-      const dimensions = entries.map(entry => entry.resource);
-      return {
-        heights: dimensions.filter(dimension =>
-          dimension.code.coding.some(sys => sys.code === HEIGHT_CONCEPT)
-        ),
-        weights: dimensions.filter(dimension =>
-          dimension.code.coding.some(sys => sys.code === WEIGHT_CONCEPT)
-        )
-      };
+      if (entries) {
```

Comment:
Should the function be explicitly typed as `Observable<Object | Void>`, or does typescript figure it out?